### ISSUE TITLE
Rancher Desktop relies on reg.exe

### ISSUE THREAD

**Issue by mar1ged** [rancher-sandbox/rancher-desktop#1071]
### Rancher Desktop Version

0.6.1

### Rancher Desktop K8s Version

(none, because I'm not even able to choose)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 Enterprise 20H2

### What CPU architecture are you using?

x64

### Windows User Only

No VPN on that machine, proxy for Internet Access, bunch of other "be(a)st-of-breed enterprise management software"

### Actual Behavior

From looking at the background log I can see that the application relies on executing reg.exe. Unfortunately this is blocked on our enterprise machines (regedit is allowed, but executing reg.exe is forbidden even for those with administrative privileges)

(I checked the logs because Rancher Desktop doesn't even start or display something on screen)

### Steps to Reproduce

Start Rancher Desktop executable

### Result

This is what I can see in the log (user redacted):

[background.log](https://github.com/rancher-sandbox/rancher-desktop/files/7659826/background.log)

### Expected Behavior

no errors ;-)

**Comment by mattfarina** [comment]
@mar1ged thank you for filing this issue. We weren't aware that some corporate environments lock down this capability. We will have to make some adjustments.

**Comment by ericpromislow** [comment]
This code was written when we moved the location of the VM image file `ext4.vhdx` from 
`%LOCALAPPDATA%/xdg.state/rancher-desktop/distro/` to 
`%LOCALAPPDATA%/rancher-desktop//distro/`

If you are migrating from version 0.4.1 or earlier, you can do this manually by ensuring the `rancher-desktop` distro is unregistered, and moving the `ext4.vhdx` file from the `xdg.state` directory to the `xdg.state`less directory. Or just delete it, and rancher-desktop will start it on the next run.

I don't see where there's any loss of functionality here. The line is written to the `background.log` file, but Rancher Desktop should continue to run. Is this not the case? Like does it just stop with the supplied message in the log file?

**Comment by mar1ged** [comment]
@ericpromislow Rancher Desktop doesn't work at all. I have an icon for the App but when I click on it I can't even see anything on screen. This can be due to this issue, the other one I created (1072) or even some other reason.

**Comment by ericpromislow** [comment]
Is your machine locked down in anyway by a corporate IT management system?

**Comment by gaktive** [comment]
@mar1ged did you try the most recent Github bundle or the existing version? Please try with https://github.com/rancher-sandbox/rancher-desktop/suites/5225089566/artifacts/160272277 and let us know if that works.

If you still have problems, please open a new ticket and link back here for context.

**Comment by mar1ged** [comment]
@gaktive The link didn't work for me but I tried 1.0.1 from the release section. Another problem (https://github.com/rancher-sandbox/rancher-desktop/issues/1072) persists, but I can no longer see any issues with reg.exe in the log.
